File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples42.vb
' <snippetlinqexamples42>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_pagingsort2 = ( _
      From c In svcContext.ContactSet _
      Where c.LastName IsNot "Parker" _
      Order By c.FirstName Descending _
      Select New With {Key c.FirstName}).Skip(2).Take(2)
  For Each c In query_pagingsort2
   Console.WriteLine(c.FirstName)
  Next c
 End Using

' </snippetlinqexamples42>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples42.cs
// <snippetlinqexamples42>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_pagingsort2 = (from c in svcContext.ContactSet
                           where c.LastName != "Parker"
                           orderby c.FirstName descending
                           select new { c.FirstName }).Skip(2).Take(2);
  foreach (var c in query_pagingsort2)
  {
   System.Console.WriteLine(c.FirstName);
  }
 }

// </snippetlinqexamples42>